gapic/admin_client.go

```diff
@@ -360,6 +417,75 @@ func (c *adminGRPCClient) DeleteProject(ctx context.Context, req *rpcpb.DeletePr
 	return err
 }
 
+// MigrateDatabaseOperation manages a long-running operation from MigrateDatabase.
+type MigrateDatabaseOperation struct {
+	lro *longrunning.Operation
+}
+
+// MigrateDatabaseOperation returns a new MigrateDatabaseOperation from a given name.
+// The name must be that of a previously created MigrateDatabaseOperation, possibly from a different process.
+func (c *adminGRPCClient) MigrateDatabaseOperation(name string) *MigrateDatabaseOperation {
+	return &MigrateDatabaseOperation{
+		lro: longrunning.InternalNewOperation(*c.LROClient, &longrunningpb.Operation{Name: name}),
+	}
+}
+
+// Wait blocks until the long-running operation is completed, returning the response and any errors encountered.
```

Comment:
Minor; Might be better to include `timeout` here as well, e.g. Wait blocks until the long-running operation is completed or timeout reached.

Reply:
Like the CLI, this is automatically-generated code (https://github.com/googleapis/gapic-generator) - I'm not sure what configuration options exist for that.

Reply:
I see; didn't noticed it was auto-generated